modules/pricer/src/test/java/com/opengamma/strata/pricer/rate/swaption/SwaptionVolatilityDataSets.java

```diff
@@ -67,6 +77,9 @@
   public static final NormalVolatilityExpiryTenorSwaptionProvider NORMAL_VOL_SWAPTION_PROVIDER_USD_STD = 
       NormalVolatilityExpiryTenorSwaptionProvider.of(SURFACE_STD, USD_1Y_LIBOR3M, DayCounts.ACT_365F, 
           VALUATION_DATE_STD, VALUATION_TIME_STD, VALUATION_ZONE_STD);
+  public static final BlackVolatilityExpiryTenorSwaptionProvider BLACK_VOL_SWAPTION_PROVIDER_USD_STD =
```

Comment:
For Black volatilities swaptions, the numbers should be more realistic, something like 20 to 50%. A BLACK_VOL array should be creatred and the SURFACE_STD should have 2 versions, one for Normal and one for Black